solidity/dashboard/src/sagas/coverage-pool.js

```diff
@@ -352,28 +325,71 @@ export function* subscribeToWithdrawalCompletedEvent() {
       payload: { event },
     } = yield take(requestChan)
     const {
-      returnValues: { underwriter, amount },
+      returnValues: { underwriter, amount, covAmount },
     } = event
     const address = yield select(selectors.getUserAddress)
 
-    if (isSameEthAddress(address, underwriter)) {
+    const { covTotalSupply, covBalance } = yield select(
+      selectors.getCoveragePool
+    )
+    const updatedCovTotalSupply = sub(covTotalSupply, amount)
```

Comment:
Should we subtract `covAmount` here from the `covTotalSupply`? I think `amount` is in KEEPs

Reply:
Please see 16271a4